src/utils/request-id.mjs

```diff
@@ -0,0 +1,6 @@
+// @ts-check
+import { v4 as generateUUID } from 'uuid'
+
+// Transform a v4 UUID to match the format used in production — alphanumeric
+// characters, all uppercase, 26 characters.
+export const generateRequestID = () => generateUUID().replace(/-/g, '').toUpperCase().slice(0, 26)
```

Comment:
if we wanted, we *could* use https://github.com/ulid/javascript to get the very same format as is being used in production, so if any of our users rely on the characteristics of ULIDs (e.g. the embedded timestamp), that'd make it easier to test in dev. I don't think anybody does that, nor that anybody *should*, so it's probably right to use UUID here. And it sames some NPM dead weight.

Reply:
Yeah I was torn on this myself. But if the idea is to bring parity with production as closely as possible, we might as well use the same format. Updated in https://github.com/netlify/cli/pull/5490/commits/d0227f71455c0aa8291cd3717760ebb590fc510f.